server/src/modules/users/application/use-cases/create-user/create-user-use-case.ts

```diff
@@ -60,47 +54,41 @@ export class CreateUserUseCase implements UseCaseWithDTO<CreateUserDTO, CreateUs
     try {
       const userAlreadyExists = await this.userRepo.exists(email)
 
-      if (userAlreadyExists && userAlreadyExists.isOk()){
+      if (userAlreadyExists && userAlreadyExists.isOk()) {
         return Result.err(new CreateUserErrors.EmailAlreadyExistsError(email.value))
       }
-        
+
       const userResult = User.create({
         email,
         password,
+        emailVerified: false,
+        isDeleted: false,
       })
       if (userResult.isErr()) return userResult
 
       const user = userResult.value
       await this.userRepo.save(user)
-      const updatedUser = await this.userRepo.getUserByUserEmail(email);
-      if(updatedUser.isErr())
+      const updatedUser = await this.userRepo.getUserByUserEmail(email)
+      if (updatedUser.isErr())
         return Result.err(new AppError.UnexpectedError(updatedUser.error.message))
 
-      if(!dto.params || !dto.params.scope){
+      if (!dto.params || !dto.params.scope) {
```

Comment:
I think there was discussion in the past about spliting the use case into separate use cases, and then moving this if/else splitting into the controller to decide which use case to call

Reply:
You're right - will add that when I add the `/logout` use-case. Coming _soon_ to a commit near you.